src/Microsoft.AspNet.HttpOverrides/OverrideHeaderMiddlewareOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.HttpOverrides
+{
+    public class OverrideHeaderMiddlewareOptions
+    {
+        public OverrideHeaderMiddlewareOptions() : this(ForwardedHeaders.None)
+        {
+        }
+
+        public OverrideHeaderMiddlewareOptions(ForwardedHeaders forwardedOptions)
```

Comment:
Options classes shouldn't have constructor parameters, make the property settable instead.